src/test/java/org/snomed/snowstorm/core/data/services/TraceabilityLogServiceTest.java

```diff
@@ -127,13 +141,59 @@ public Activity getActivityWithTimeout(int maxWait) throws InterruptedException
 	void createCommitCommentRebase() {
 		Commit commit = new Commit(new Branch("MAIN/A"), Commit.CommitType.REBASE, null, null);
 		commit.setSourceBranchPath("MAIN");
-		assertEquals("kkewley performed merge of MAIN to MAIN/A", traceabilityLogService.createCommitComment("kkewley", commit, Collections.emptySet(), false));
+		assertEquals("kkewley performed merge of MAIN to MAIN/A", traceabilityLogService.createCommitComment("kkewley", commit, Collections.emptySet(), false, null));
 	}
 
 	@Test
 	void createCommitCommentPromotion() {
 		Commit commit = new Commit(new Branch("MAIN"), Commit.CommitType.PROMOTION, null, null);
 		commit.setSourceBranchPath("MAIN/A");
-		assertEquals("kkewley performed merge of MAIN/A to MAIN", traceabilityLogService.createCommitComment("kkewley", commit, Collections.emptySet(), false));
+		assertEquals("kkewley performed merge of MAIN/A to MAIN", traceabilityLogService.createCommitComment("kkewley", commit, Collections.emptySet(), false, null));
+	}
+
+	@Test
+	void testDeltaImportWithNoChanges() {
+		final ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+		listAppender.start();
+		((Logger) LoggerFactory.getLogger(TraceabilityLogService.class)).addAppender(listAppender);
+		traceabilityLogService.setEnabled(true);
+		final Commit commit = new Commit(new Branch("MAIN/Delta"), Commit.CommitType.CONTENT, null, null);
+		commit.setSourceBranchPath("MAIN");
+		final Branch branch = commit.getBranch();
+		branch.setMetadata(ImmutableMap.of("importType", "Delta"));
+		final PersistedComponents persistedComponents = new PersistedComponents();
+		traceabilityLogService.preCommitCompletion(commit);
+		final List<ILoggingEvent> logsList = listAppender.list;
+		assertEquals("Skipping traceability because there was no traceable change.", logsList.get(0).getMessage());
+		assertEquals(Level.INFO, logsList.get(0).getLevel());
+		listAppender.stop();
+	}
+
+	@Test
+	void testDeltaImportWithOneAdditionChange() {
+		activitiesLogged.clear();
+		final Commit commit = new Commit(branchService.create("MAIN/RF2DeltaImport"), Commit.CommitType.CONTENT, null, null);
+		final PersistedComponents persistedComponents =
+				PersistedComponents.newBuilder()
+								   .withPersistedConcepts(Collections.singleton(new Concept("3311481044").addFSN("Test FSN")))
+								   .withPersistedDescriptions(Collections.singleton(new Description("8635753033", 1, true, "900000000000012033", "3311481044",
+																									"en", "900000000000013044", "Test term", "900000000000448022"))).build();
+		traceabilityLogService.logActivity(null, commit, persistedComponents, false, "Delta");
+		activitiesLogged.stream().map(activity -> activity.getCommitComment().contains("RF2 Import - Updating concept Test FSN")).forEach(Assert::assertTrue);
```

Comment:
Nice assertion including the new message prefix.
Be careful with expectations on collections, if no activities are logged this will still pass..

Reply:
Fixed